src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderIdentifier.java

```diff
@@ -45,17 +49,6 @@ public class OffenderIdentifier {
     @Schema(description = "Creation date and time", example = "2018-01-21 15:00:00")
     private LocalDateTime whenCreated;
 
-    public OffenderIdentifier(@NotBlank String type, @NotBlank String value, String offenderNo, Long bookingId, String issuedAuthorityText, LocalDate issuedDate, String caseloadType, LocalDateTime whenCreated) {
-        this.type = type;
-        this.value = value;
-        this.offenderNo = offenderNo;
-        this.bookingId = bookingId;
-        this.issuedAuthorityText = issuedAuthorityText;
-        this.issuedDate = issuedDate;
-        this.caseloadType = caseloadType;
-        this.whenCreated = whenCreated;
-    }
-
-    public OffenderIdentifier() {
-    }
+    @Schema(description = "Offender ID", example = "234547")
+    private Long offenderId;
```

Comment:
Add offenderId to offenderIdentifier so we know if it relates to the main offender or an alias

Reply:
would have been nicer to convert the file to kotlin 😁